src/controllers/handlers/about-handler.ts

```diff
@@ -31,14 +31,15 @@ export type About = {
 // handlers arguments only type what they need, to make unit testing easier
 export async function aboutHandler(
   context: Pick<
-    HandlerContextWithPath<'serviceDiscovery' | 'status' | 'realm' | 'config' | 'rpcSessions', '/about'>,
+    HandlerContextWithPath<'serviceDiscovery' | 'status' | 'realm' | 'config' | 'rpcSessions' | 'metrics', '/about'>,
     'url' | 'components'
   >
 ) {
-  const { realm, config, status, serviceDiscovery, rpcSessions } = context.components
+  const { realm, config, status, serviceDiscovery, rpcSessions, metrics } = context.components
   const commsProtocol = await config.requireString('COMMS_PROTOCOL')
 
   const userCount = rpcSessions.sessions.size
+  metrics.observe('explorer_bff_connected_users', {}, userCount)
```

Comment:
perhaps https://github.com/decentraland/explorer-bff/blob/13966ad9dd0b8b1fd53857ac6570182411ec71c8/src/controllers/rpc/initialize-rpc-server-handler.ts#L58 is a better way place to track this, otherwise we will only track users if this endpoint is called

Reply:
🤦‍♂️  thanks!